src/test/java/nextstep/security/filter/converter/LoginFormAuthenticationConverterTest.java

```diff
@@ -0,0 +1,75 @@
+package nextstep.security.filter.converter;
+
+import nextstep.security.authentication.Authentication;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LoginFormAuthenticationConverterTest {
+
+    private LoginFormAuthenticationConverter loginFormAuthenticationConverter;
+
+    @BeforeEach
+    void init() {
+        loginFormAuthenticationConverter = new LoginFormAuthenticationConverter();
+    }
```

Comment:
```suggestion
    private final LoginFormAuthenticationConverter loginFormAuthenticationConverter = new LoginFormAuthenticationConverter();
```
junit5에서는 기본 테스트 초기화가 메소드 단위이기 때문에 별도로 메소드로 분리하지 않으셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle